governance/executive-council.md

```diff
@@ -0,0 +1,262 @@
+
+Lazy consensus means that decisions are approved by default unless a member of
+the Executive Council explicitly raises an objection within a predefined period.
+This allows decisions to move forward efficiently while still providing an
+opportunity for discussion. The proposal is accepted if no objections are raised
+within the specified timeframe.
+
+When objections are raised, further discussion occurs, and a formal vote is held
+if needed. This ensures decisions reflect the collective agreement of the Council
+while minimizing unnecessary delays.
+
+:::
+
+
+All decisions made by the Executive Council must align with the legal
+obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityin.org/).
+This ensures compliance with overarching governance structures and financial
+responsibilities.
+
+### Decisions requiring an Executive Council vote
+
+The Executive Council (EC) defines and directs pyOpenSci’s strategy and major
+organizational direction. It is responsible for approving significant decisions
+that strongly impact pyOpenSci’s strategy, governance, or financial well-being.
+Generally, items with legal or fiduciary implications require a formal vote,
+with the outcome documented in our [meeting minutes](ec-minutes).
+
+The EC will vote on the following:
+
+- **Annual budget**: Approval of the organization's yearly budget.
+- **Mission, vision, and values**: Decisions related to pyOpenSci’s core principles.
+- **Governance and policy changes**: Approval of changes to governance structures,
+  including updates to the Code of Conduct.
+- **Significant budget deviations**: Decisions involving major deviations from the
+  approved annual budget.
+- **Financial health implications**: Any decision with substantial implications for
+  pyOpenSci’s financial stability.
+- **Strategic direction**: Approval of major changes to organizational strategy and
+  development roadmaps.
+- **Executive director’s salary**: The EC independently determines the salary of the
+  Executive Director.
+
+:::note
+The Executive Director is authorized to make all other decisions. However, the
+Executive Council serves as a resource for the Executive Director and may provide
+feedback and input upon request.
+:::
+
+## Collaboration between the Executive Director and the Executive Council
+
+While not requiring a formal vote, the Executive Director may request feedback
+from the Executive Council on significant or public-facing decisions. These discussions should
+occur in the **executive-council** Slack channel to ensure transparency and
+collaborative input. Examples include:
+
+- Significant public-facing changes to the pyOpenSci website.
+- Decisions impacting staff or human resources (e.g., hiring or termination).
+- Operational policies for staff (unless they carry major financial implications).
+- Strategic decisions about pursuing proposals or writing grants.
+- Substantial changes to pyOpenSci programs.
+
```

Comment:
Add here about lazy consensus approach

Reply:
great - i've reorganized the section based on your suggestions @tracykteal i just pushed up the changes. 🚀